templates/InstallChocolatey.ps1.erb

```diff
@@ -100,10 +100,14 @@ if ($unzipMethod -eq '7zip') {
   Write-Output "Extracting $file to $tempDir..."
   Start-Process "$7zaExe" -ArgumentList "x -o`"$tempDir`" -y `"$file`"" -Wait -NoNewWindow
 } else {
-  $shellApplication = new-object -com shell.application
-  $zipPackage = $shellApplication.NameSpace($file)
-  $destinationFolder = $shellApplication.NameSpace($tempDir)
-  $destinationFolder.CopyHere($zipPackage.Items(),0x10)
+  if ($PSVersionTable.PSversion.major -lt 5) {
+    $shellApplication = new-object -com shell.application

```

Comment:
I actually never tested the assembly part on PS < 4, but it was a new feature in .NET2.0:
https://msdn.microsoft.com/en-us/library/system.io.compression(v=vs.80).aspx

The `Expand-Archive` - commandlet is a new (and long anticipated) feature of PS 5.